src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -614,12 +619,22 @@
         {
             "name": "codewhisperer_codeScanIssueHover",
             "description": "Called when a code scan issue is hovered over",
-            "metadata": [{ "type": "findingId" }, { "type": "detectorId" }]
+            "metadata": [{ "type": "findingId" }, { "type": "detectorId" }, { "type": "ruleId", "required": false }]
         },
         {
             "name": "codewhisperer_codeScanIssueApplyFix",
             "description": "Called when a code scan issue suggested fix is applied",
-            "metadata": [{ "type": "findingId" }, { "type": "detectorId" }, { "type": "component" }]
+            "metadata": [
+                { "type": "findingId" },
+                { "type": "detectorId" },
+                { "type": "component" },
+                { "type": "ruleId", "required": false }
+            ]
+        },
+        {
+            "name": "codewhisperer_codeScanIssueViewDetails",
```

Comment:
why isn't this being added to the common definitions? https://github.com/aws/aws-toolkit-common/blob/main/telemetry/definitions/commonDefinitions.json